ocs_ci/ocs/ui/base_ui.py

```diff
@@ -830,6 +830,8 @@ def login_ui(console_url=None):
     wait = WebDriverWait(driver, 60)
     driver.maximize_window()
     driver.get(console_url)
+    # Validate proceeding to the login console before taking any action:
+    proceed_to_login_console(driver)
```

Comment:
@jimdevops19 , I think we don't need any change in login_ui, instead:https://github.com/red-hat-storage/ocs-ci/blob/f57e12c4049c409570a9103b8807c82e5630e1bf/ocs_ci/ocs/acm/acm.py#L280 , add here .